### PR TITLE
Update dependencies to support NO_PROXY

### PR DESCRIPTION
**Pull request by giuliov**
We found that the version of libraries used had no support for NO_PROXY, which is crucial for our environments.

### CODE REVIEW

File: src/WorkItemUpdater/WorkItemUpdater.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import tl = require('vsts-task-lib/task');
```
Comment (nerdomancer) [inline src/WorkItemUpdater/WorkItemUpdater.ts:1]: This package is also deprecated. Would you like to update it as well?
This is the replacement for it [azure-pipelines-task-lib](https://www.npmjs.com/package/azure-pipelines-task-lib)

Comment (nerdomancer) [reply src/WorkItemUpdater/WorkItemUpdater.ts:1]: I see that `package.json` is updated, but maybe this line was forgotten to be updated.

Comment (giuliov) [reply src/WorkItemUpdater/WorkItemUpdater.ts:1]: can't remember, as long as it works with Azure DevOps on premise, I'm fine

Comment (BlueBasher) [reply src/WorkItemUpdater/WorkItemUpdater.ts]: It should indeed be updated as well.

**Pull request merged**